20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+The first section of the report titled "Unclaimed — jump right in!" is usually a good place to find tickets that have been newly given the "good-first-bug" keyword, and can cover all kinds of development, testing and documentation needs. Sometimes tickets look as though they are well under way, but often they simply need their patches refreshed which is generally an easy thing to tackle (see [Applying a Patch][] and [Creating a Patch][]).
+
+The introduction to this report has some very handy links to areas of the [Core Contributor Handbook](https://make.wordpress.org/core/handbook/) that I urge you to read. The [WordPress Coding Standards](https://make.wordpress.org/core/handbook/coding-standards/) are a must read before you start development, and hopefully you're already familiar with the [WordPress Codebase](https://make.wordpress.org/core/handbook/the-wordpress-codebase/). It's worth ready the ["how we use our bug tracker"](https://make.wordpress.org/core/handbook/trac/) article to get familiar with the terminology and process flow.
+
+### No Patch
+[https://core.trac.wordpress.org/tickets/no-patch](https://core.trac.wordpress.org/tickets/no-patch)
+
+The "No Patch" or "Needs patch, but has none" report is great for finding tickets to work on that still need a patch. It could be that no-one has tackled the ticket yet or that an existing patch has problems and a new clean patch is required.
+
+There is usually a "Next Release" section to this report that I encourage you to tackle tickets from if you want to try and get some "props" sooner rather than later, as tickets on that list are naturally more likely to be reviewed and tested quicker as they have been prioritised for the next release. It's always a good feeling to see your name mentioned in one of the [WordPress Core Weekly](https://make.wordpress.org/core/tag/week-in-core/) posts, and will encourage you to tackle more tickets in the future.
+
+### Needs Unit Tests
+[https://core.trac.wordpress.org/tickets/needs-unit-tests](https://core.trac.wordpress.org/tickets/needs-unit-tests)
+
+Are you handy with Unit Tests? Maybe you're looking to explore WordPress Core and contribute at the same time? Then the "Needs Unit Tests" report is just the ticket!
+
+### Next Major and Next Minor Release
+[https://core.trac.wordpress.org/tickets/major/workflow](https://core.trac.wordpress.org/tickets/major/workflow)
+[https://core.trac.wordpress.org/tickets/minor/workflow](https://core.trac.wordpress.org/tickets/minor/workflow)
+
+These catch-all reports are a great way to target your efforts on tickets assigned to the next couple of releases. The above links are versions of the [Next Major](https://core.trac.wordpress.org/tickets/major) and and [Next Minor](https://core.trac.wordpress.org/tickets/minor) Release reports grouped by Workflow Status, I prefer these as it is much easier to find tickets that match your skills (development, testing etc).
```

Comment:
@polevaultweb 
> Too many capitals here; 'Next Major', 'Release', 'Workflow Status'

I don't see an alternative as they are the names for the reports and grouping?

> The above links are versions of the Next Major and and Next Minor Release reports grouped by Workflow Status which I prefer, as it is much easier to find tickets that match your skills (development, testing etc).

This feels disjointed to me, and not conversational.